code/modules/research/designs/boards/machine_science.dm

```diff
@@ -112,3 +112,13 @@
 	materials = list(MAT_GLASS = 2000, SACID = 20)
 	category = "Console Boards"
 	build_path = /obj/item/weapon/circuitboard/suspension_gen
+
+/datum/design/weathercontroldevice
```

Comment:
I feel like the design should be exclusive to the map considering it has no utility outside of the specific map. I think you throw it in the station name's .dm (?)

Reply:
It just doesn't turn on if the map has no climate. Leaving it available is more modular in case anyone ever makes another map with a climate. The whole system for climates is modular and the weather control system is no exception: as long as you order your weathers properly you can use this with any map